src/utils/tracking/googleAnalytics/index.tsx

```diff
@@ -0,0 +1,28 @@
+import * as gtag from '@/libs/gtag';
+import Script from 'next/script';
+function GoogleAnalytics() {
+  return (
+    <>
+      <Script
+        async
+        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_KEY}`}
+      ></Script>
```

Comment:
```suggestion
      <Script
        async
        src={`https://www.googletagmanager.com/gtag/js?id=${gtag.GA_KEY}`}
      />
```
단일 태그 사용으로 통일하면 좋을 것 같아요!

Reply:
넵 좋은거같네요 수정할게용